src/WebJobs.Script.WebHost/Middleware/ExceptionMiddleware.Log.cs

```diff
@@ -11,14 +11,19 @@ internal partial class ExceptionMiddleware
         private static class Logger
         {
             private static readonly Action<ILogger, Exception> _responseStarted =
-               LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(ResponseStarted)), "The response has already started, the status code will not be modified.");
+                LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(ResponseStarted)), "The response has already started, the status code will not be modified.");
```

Comment:
```suggestion
                LoggerMessage.Define(LogLevel.Debug, new EventId(1, nameof(ResponseStarted)), "The response has already started. The status code will not be modified.");
```

Reply:
I don't want to edit the log message since I didn't make any changes here